core/src/main/scala/tech/beshu/ror/configuration/EsConfig.scala

```diff
@@ -38,7 +39,7 @@ object EsConfig {
           (implicit envVarsProvider:EnvVarsProvider): Task[Either[LoadEsConfigError, EsConfig]] = {
     val configFile = File(s"${esConfigFolderPath.toAbsolutePath}/elasticsearch.yml")
     (for {
-      _ <- EitherT.fromEither[Task](Either.cond(configFile.exists, (), FileNotFound(configFile)))
+      _ <- EitherT.fromEither[Task](Either.cond(doPrivileged(configFile.exists), (), FileNotFound(configFile)))
```

Comment:
I'm little bit concerned that we should add `doPrivileged` in so many places. What about creating `File` adapter (I wanted to write decorator, but there is no sense to expose all methods from `better.file.File` class) and encapsulate `doPrivileged` in it?